packages/bpk-component-navigation-bar/src/BpkNavigationBar.tsx

```diff
@@ -100,7 +100,7 @@ const BpkNavigationBar = (props: Props) => {
           </BpkText>
         </span>
       ) : (
-        <div className={getClassNames('bpk-navigation-bar__title-container')}>{title}</div>
```

Comment:
Why not let the consumer passing the title include their own styling if it's just text then it will be using the default above.